packages/fluentui/react-northstar/.eslintrc.json

```diff
@@ -10,9 +10,13 @@
     },
     {
       "files": "src/**/*.{ts,tsx}",
+      "parserOptions": {
+        "lib": ["es2015", "dom"]
```

Comment:
This is now necessary to avoid warnings about DOM types such as `ClientRect` being undefined. Similar with the `JSX` change below.